### ISSUE TITLE
Local build datepicker.scss error

### ISSUE THREAD

**Issue by delai** [Hacker0x01/react-datepicker#461]
hey, great component. But there is a build error in my here. How can i fix it?

my node version is v4.2.1

the error message: 

ERROR in ./src/stylesheets/datepicker.scss
Module build failed: Error: dlopen(/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/node-sass/vendor/darwin-x64-46/binding.node, 1): no suitable image found.  Did find:
	/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/node-sass/vendor/darwin-x64-46/binding.node: truncated mach-o error: segment __TEXT extends to 1212416 which is past end of file 1165872
    at Error (native)
    at Object.Module._extensions..node (module.js:460:18)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/node-sass/lib/index.js:23:15)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/sass-loader/index.js:4:12)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
 @ ./docs-site/src/example_components.jsx 87:0-53

ERROR in ./docs-site/src/style.scss
Module build failed: Error: dlopen(/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/node-sass/vendor/darwin-x64-46/binding.node, 1): no suitable image found.  Did find:
	/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/node-sass/vendor/darwin-x64-46/binding.node: truncated mach-o error: segment __TEXT extends to 1212416 which is past end of file 1165872
    at Error (native)
    at Object.Module._extensions..node (module.js:460:18)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/node-sass/lib/index.js:23:15)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/sass-loader/index.js:4:12)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
 @ ./docs-site/src/example_components.jsx 89:0-23
Child extract-text-webpack-plugin:
    
    ERROR in dlopen(/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/node-sass/vendor/darwin-x64-46/binding.node, 1): no suitable image found.  Did find:
    	/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/node-sass/vendor/darwin-x64-46/binding.node: truncated mach-o error: segment __TEXT extends to 1212416 which is past end of file 1165872
Child extract-text-webpack-plugin:
    
    ERROR in dlopen(/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/node-sass/vendor/darwin-x64-46/binding.node, 1): no suitable image found.  Did find:
    	/Users/arthasshih/FrontEnd/React/react-datepicker/node_modules/node-sass/vendor/darwin-x64-46/binding.node: truncated mach-o error: segment __TEXT extends to 1212416 which is past end of file 1165872
Warning: Task "webpack:docs" failed. Use --force to continue.

Aborted due to warnings.

**Comment by rafeememon** [comment]
Google says: https://paulschreiber.com/blog/2015/06/22/fixing-dlopen-no-suitable-image-found-errors-with-node-sass-and-grunt/

tl;dr reinstalling node-sass will probably fix it
